src/main/cpp/subsystems/LimelightSubsystem.cpp

```diff
@@ -143,9 +146,10 @@ double LimelightSubsystem::GetApriltagShooterTheta(double dist, double angleTrim
     if(dist != 0.0)
     {
         frc::SmartDashboard::PutNumber("Distance AprilTag", dist);
-        // return -0.2351* pow((dist+angleTrim),3) + 4.38 * pow((dist+angleTrim), 2) - 29 * (dist+angleTrim) + 89.64;
-        return -0.2238* pow((dist+angleTrim),3) + 4.1431 * pow((dist+angleTrim), 2) - 27.38 * (dist+angleTrim) + 89.093;
-        // y = -0.2238x3 + 4.1431x2 - 27.38x + 89.093
+        // return -0.2351* pow((dist+angleTrim),3) + 4.38 * pow((dist+angleTrim), 2) - 29 * (dist+angleTrim) + 89.64; //initial curve not using the PID shooter 
+        // return -1.556* pow((dist+angleTrim),3) + 17.162 * pow((dist+angleTrim), 2) - 68 * (dist+angleTrim) + 127.2; //power of 3 magnolia
+        return 0.6042 * pow((dist+angleTrim),4) - 8.94 * pow((dist+angleTrim),3) + 49.7 * pow((dist+angleTrim),2) - 129.06 * (dist+angleTrim) + 168.18;
```

Comment:
Is this going to work correctly with order of operations?

Reply:
I believe so, it's what we have been running all day